### DISCUSSION TITLE
Getting Maximum call stack size exceeded error when running .count()

### DISCUSSION THREAD

**Discussion by FedeDevoto** [mikro-orm/mikro-orm#3722] [Q&A]
Hi all, 

Running em.count() I'm getting Maximum call stack size exceeded in the version v5.5.3 but it does work in v5.3.0. 

Entity
`@Entity({ tableName: 'studies' })
export class Studies {
    @Type(() => String)
    @PrimaryKey({ columnType: 'bigint' })
    id!: string;

    @Type(() => Date)
    @Property()
    createdDate!: Date;

    @Type(() => String)
    @Property({ columnType: 'bigint' })
    createdByUserId!: string;

    @Type(() => Date)
    @Property({ nullable: true })
    modifiedDate?: Date;

    @Type(() => String)
    @Property({ columnType: 'bigint', nullable: true })
    modifiedByUserId?: string;

    @Property({ length: 255 })
    state!: string;

    @Property({ length: 255 })
    editState!: string;

    @Property({ version: true, columnType: 'bigint', default: 0 })
    jpaVersion!: number;

    @Property({ length: 1024 })
    name!: string;

    @Property({ length: 255, nullable: true, default: null })
    datasetName?: string;

    @Property({ columnType: 'char(36)', length: 36, nullable: true, default: null })
    datasetGuid?: string;

    @Property({ type: 'json', nullable: true })
    populationDefinition?: PopulationDefinitionDTO;

    @Property({ type: 'json', nullable: true })
    outcomes?: OutcomesDto;
}`

Constructor
@InjectRepository(Studies) private readonly studiesRepository: EntityRepository<Studies>,

Function
this.studiesRepository.count(filterQuery) // where filterQuery is an object {status: 'active'}

Does something change? I would appreciate your help!

Thanks

**Comment by B4nan** [comment]
Try latest dev version, this should be already fixed in master.

**Reply by FedeDevoto** [reply]
5.5.3?

**Reply by B4nan** [reply]
i said dev version, not stable
<img width="266" alt="image" src="https://user-images.githubusercontent.com/615580/201325707-37511686-32c6-480c-8a30-a2b27c931f09.png">

**Reply by FedeDevoto** [reply]
thanks! it works. May I ask you when is planned to release the new version?

**Reply by julestruong** [reply]
hi @B4nan , can you explain what is going on , under the hood ? we might help if you need
